### ISSUE TITLE
Script broken on Arch Linux

### ISSUE THREAD

**Issue by davigamer987** [hoverbike1/lazy-pack-generator#2]
I tried running the .bin and the .zip version, both result in this error:

❯ ./lazy_pack.bin
Traceback (most recent call last):
  File "Lazy_Pack.py", line 3, in <module>
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "PyInstaller/loader/pyimod02_importers.py", line 385, in exec_module
  File "tkinter.py", line 11, in <module>
FileNotFoundError: [Errno 2] No such file or directory: '/home/davi/.local/share/yuzu/load/0100F2C0115B6000/Linux Pack Generator/lib64/python3.9/_import_failed/import_failed.map'
[10533] Failed to execute script 'Lazy_Pack' due to unhandled exception!

Using Python 3.11.3, As a temporary workaround using the Windows version through wine works fine

**Comment by hoverbike1** [comment]
try running pip install tkinter in terminal? seems like its complaining about missing the tkinter library?

**Comment by mr-sihc** [comment]
nope, same error
its also failing IN tkinter (more speicifically line 10)
so it IS installed

**Comment by hoverbike1** [comment]
https://pixeldrain.com/u/mNuBHPL3

open terminal
cd to the same directory as the py file
then "python3 Lazy_Pack.py"
does that work for you?

**Comment by mr-sihc** [comment]
that works!

**Comment by alexzhaohong** [comment]
Hi hoverbike1, This worked for me as well, as I am also using a Steam Deck. I couldn't find directions on how to use the .bin file in "Linux Lazy Pack" in the 2.5.3 release

**Comment by hoverbike1** [comment]
yeah, that bin file was a desperate attempt from me to put something out that worked on linux, however, i haven't a clue about linux, so that went as well as i couldve expected.

i guess i'll need to provide these source files instead - should at least work for those that know how to run it, i guess.
thanks for the feedback
